201-redis-vnet-geo-replication/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+    "type": "QuickStart",
+    "itemDisplayName": "Create two Caches with Virtual Network and Geo-Replication",
```

Comment:
Use:

Create two Redis Caches...

Reply:
Due to branding requirements, this cannot be changed to "Redis Caches." Alternatively we could use "Azure Cache for Redis" however we run into character length limitations. To avoid this, I will use "caches" as-is.